modulefiles/module_setup.hera.lua

```diff
@@ -0,0 +1,14 @@
+help([[
+Load environment to run GFS workflow setup scripts on Hera
+]])
+
+-- Temporary until official hpc-stack is updated
+
+load(pathJoin("rocoto"))
+
+prepend_path("MODULEPATH", "/scratch2/NCEPDEV/ensemble/save/Walter.Kolczynski/hpc-stack/modulefiles/stack")
+load(pathJoin("hpc", "1.2.0"))
+load(pathJoin("hpc-miniconda3", "4.6.14"))
+load(pathJoin("ufswm", "1.0.0"))
```

Comment:
Load `gfs_workflow` instead of `ufswm` and you won't have to `pip install yaml`.

Reply:
oh ok.